File: samples/client/3_0_3_unit_test/java/src/main/java/org/openapijsonschematools/client/components/schemas/NumberTypeMatchesNumbers.java
package org.openapijsonschematools.client.components.schemas;
import org.checkerframework.checker.nullness.qual.Nullable;
import org.openapijsonschematools.client.schemas.NumberJsonSchema;

public class NumberTypeMatchesNumbers extends NumberJsonSchema {
    // nest classes so all schemas and input/output classes can be public
    
    
    public static class NumberTypeMatchesNumbers1 extends NumberJsonSchema.NumberJsonSchema1 {
        private static @Nullable NumberTypeMatchesNumbers1 instance = null;
        public static NumberTypeMatchesNumbers1 getInstance() {
            if (instance == null) {
                instance = new NumberTypeMatchesNumbers1();
            }
            return instance;
        }
    }

}

File: samples/client/3_0_3_unit_test/kotlin/src/main/kotlin/org/openapijsonschematools/client/components/schemas/NumberTypeMatchesNumbers.kt
package org.openapijsonschematools.client.components.schemas
import org.openapijsonschematools.client.schemas.NumberJsonSchema

class NumberTypeMatchesNumbers: NumberJsonSchema() {
    // nest classes so all schemas and input/output classes can be public
    
    
    class NumberTypeMatchesNumbers1 : NumberJsonSchema.NumberJsonSchema1() {
        companion object {
            @Volatile
            private var instance: NumberTypeMatchesNumbers1? = null
    
            fun getInstance() =
                instance ?: synchronized(this) {
                    instance ?: NumberTypeMatchesNumbers1().also { instance = it }
                }
        }
    }

}
